action.yml

```diff
@@ -1,27 +1,47 @@
 name: 'Leviathan Action'
 description: 'Test your software directly on hardware using the Levaithan Testing framework'
+inputs:
+  force-build:
```

Comment:
Where would we specify force-build to be true
I was expecting leviathan images to be uploaded from the leviathan repository.